packages/core/src/asset/ResourceManager.ts

```diff
@@ -593,6 +595,7 @@ export class ResourceManager {
   initVirtualResources(config: EditorResourceItem[]): void {
     config.forEach((element) => {
       this._virtualPathMap[element.virtualPath] = element.path;
+      this._dependencyMap[element.virtualPath] = element.depndentAssets;
```

Comment:
What's the value of element.dependentAssets if asset has not dependentAssets. if it is undefined, still set?

Reply:
done